README.md

```diff
@@ -1,2 +1,12 @@
 # Operational-Planning-Tool
-That is a Cloud Infrastructure Project for a Operational Planning of Companies
+### That is a Cloud Infrastructure Project for a Operational Planning of Companies
+
+This project is intended for better planning in craft businesses by comparing working times, sick leave and later possibly also working locations. In addition, a stock is also installed, which means that material can also be used per project and the current stock is updated. As an extra, which may still be entered, is the automatic order when there is a minimum stock and an order stop when there is a maximum stock
```

Comment:
craft business = Bastell-Business. Was wolltest du sagen, um welche Firmen es sich handeln soll? Was meinst du mit stock is installed? Ein Bestandserfasser? Lass uns da gleich gerne drüber reden, dann können wir den Text zusammen anpassen.

Reply:
Ich wollte damit sagen das es sich um Handwerksbetriebe dreht weil das wahrscheinlich dabei das Haupteinsatzgebiet ist

Und ja ich meine eine art bestandserfasster in form einer DB